<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RPSGame</class>
 <widget class="QMainWindow" name="RPSGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RPSGame</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="rockButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Rock</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Sprites/human_rock.png</normaloff>:/Sprites/human_rock.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>54</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="paperButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Paper</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Sprites/human_paper.png</normaloff>:/Sprites/human_paper.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="scissorsButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Scissors</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Sprites/human_scissor.png</normaloff>:/Sprites/human_scissor.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="computerChoiceLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Computer chose ....</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Result Awaited ....</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="userScoreLCD">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="computerScoreLCD">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="userScoreLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>-40</y>
      <width>230</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/Sprites/score_human.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="computerScoreLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>-40</y>
      <width>230</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/Sprites/score_robot.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>470</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Sprites/button_quit.png</normaloff>:/Sprites/button_quit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labelUserSprite">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>170</width>
      <height>170</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelRoboSprite">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>170</width>
      <height>170</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelBackground">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/Sprites/blackBG.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>labelBackground</zorder>
   <zorder>rockButton</zorder>
   <zorder>paperButton</zorder>
   <zorder>scissorsButton</zorder>
   <zorder>computerChoiceLabel</zorder>
   <zorder>resultLabel</zorder>
   <zorder>userScoreLCD</zorder>
   <zorder>computerScoreLCD</zorder>
   <zorder>userScoreLabel</zorder>
   <zorder>computerScoreLabel</zorder>
   <zorder>quitButton</zorder>
   <zorder>labelUserSprite</zorder>
   <zorder>labelRoboSprite</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>RPSGame</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>528</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>564</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
